doc/modules/linear_model.rst

```diff
@@ -266,6 +266,11 @@ They also tend to break when the problem is badly conditioned
 
   * :ref:`sphx_glr_auto_examples_linear_model_plot_lasso_model_selection.py`
 
+.. note:: **Comparison with the regularization parameter of SVM**
```

Comment:
Added the change mentioning the two values of C, but am skeptical about mentioning it at the top. Please let me know what you feel.